interface.go

```diff
@@ -67,6 +67,9 @@ type Config struct {
 	// If not set, it uses all versions available.
 	// Warning: This API should not be considered stable and will change soon.
 	Versions []protocol.VersionNumber
+	// Ask the server to truncate the connection ID sent in the Public Header.
+	// Only valid for the client.
+	TruncateConnectionID bool
```

Comment:
Yeah, that's what I wrote in my PR. Would you be ok with `RequestConnectionIDTruncation`? It's a bit too long for my taste, but tbh, I don't have any better idea.